File: lab-9/subplot.py
import numpy as np
import matplotlib.pyplot as plt

t = np.arange(0, 10.01, 0.01)
x = np.sin(0.5 * np.pi * t)
y = np.sin(0.5 * np.pi * t)
z = x + y

plt.figure()

plt.subplot(2, 3, 1)
plt.plot(x)

plt.subplot(2, 3, 2)
plt.plot(y)

plt.subplot(2, 3, 3)
plt.plot(z)

plt.show()

File: lab-9/subplot.m
t=0:0.01:10
x=sin(0.5*pi*t)
y=cos(0.5*pi*t)
z=x+y
figure()
subplot(2,3,1)
plot(x)
subplot(2,3,2)
plot(y)
subplot(2,3,3)
plot(z)
